### DISCUSSION TITLE
Option for old segemented slider style

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#27323] [Ideas] [answered]
I enjoy playing with old skins occasionally, but one thing that always bugged me is that in modern clients they never really looked right, due to the sliders always having a smooth track. I think it could be nice to bring back an option to enable the old segmented slider tracks. The best way to do this would probably be to just check for the old `SliderStyle: 1` setting in the skin.ini (https://osu.ppy.sh/wiki/en/Skinning/History#.ini). This would also avoid adding yet another setting to the settings menu.

![segmented slider track](https://github.com/ppy/osu/assets/22381687/8415a232-1d31-4320-985a-20f71e5d37a7)

**Comment by peppy** [comment, marked as answer]
Not going to happen sorry

**Reply by cl8n** [reply]
is this something you'd reconsider as part of https://github.com/ppy/osu/discussions/27795 later on?

would be nice to hear your reasoning for this bc at surface level I don't see how it's much different from e.g. default skins having different slider body styles already

**Reply by peppy** [reply]
It could potentially be emulated using a texture. The original (broken) rendering style will not return verbatim.
